triple-reads-backend/index.ts

```diff
@@ -201,6 +211,183 @@ app.get('/books/:isbn', async (req: Request, res: Response) => {
   }
 });
 
+//POST TO INSERT
+app.post('/books', async (req: Request, res: Response) => {
+  try {
+    const { title, isbn, datePublished, abstract, image, authorNames, genreNames, publisher, admin, dateAdded } = req.body;
```

Comment:
Since the PR with the swagger file is merge, please have a look at the naming of the variables (or properties) in the POST book and update accordingly.